### DISCUSSION TITLE
Compile time errors in latest master micro-ui

### DISCUSSION THREAD

**Discussion by mohsinansarirepo** [egovernments/Digit-Core#156] [Bugs]
Hi @jagankumar-egov,
There are two minor compile time issues in micro-ui while running on local machine
1-Can't resolve @egovernments/digit-ui-module-hrms
2-Syntax error in index.js

![Screenshot from 2023-08-03 13-26-57](https://github.com/egovernments/Digit-Core/assets/92909304/a3278631-6726-4d4f-961a-c7a724233de2)
![Screenshot from 2023-08-03 11-55-58](https://github.com/egovernments/Digit-Core/assets/92909304/2bb8be0a-02c4-4863-ac3a-47a997db10c7)

**Comment by jagankumar-egov** [comment]
@mohsinansarirepo  
take a pull, it has been fixed in the following commit,
https://github.com/egovernments/DIGIT-OSS/pull/97.
